### DISCUSSION TITLE
Does Dify have plans to support MySQL? Thank you.

### DISCUSSION THREAD

**Discussion by joylijoy** [langgenius/dify#3815] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

There is no MySQL cloud platform component available nearby

### 2. Describe the feature you'd like to see

I hope it can support MySQL.

### 3. How will this feature improve your workflow or experience?

There is an existing instance of MySQL

### 5. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by crazywoola** [comment]
#2371 We do have a pr for this. But we have no official plan for this right now.
